docs/source/API/core/view/view.rst

```diff
@@ -36,30 +36,31 @@ Parameters
 .. |LayoutStride| replace:: :cppkokkos:func:`LayoutStride`
 
 Template parameters other than ``DataType`` are optional, but ordering is enforced.
-That means for example that ``LayoutType`` can be omitted but if both ``MemorySpace``
+That means for example that ``Layout`` can be omitted, but if both ``MemorySpace``
 and ``MemoryTraits`` are specified, ``MemorySpace`` must come before ``MemoryTraits``.
 
 * ``DataType``:
 
   Defines the fundamental scalar type of the ``View`` and its dimensionality.
-  The basic structure is ``ScalarType STARS BRACKETS`` where the number of STARS denotes
-  the number of runtime length dimensions and the number of BRACKETS defines the compile time dimensions.
-  Due to C++ type restrictions runtime dimensions must come first.
+  The basic structure is ``ScalarType number of STARS, number of BRACKETED items`` where the number of STARS denotes
```

Comment:
It's not relly the number of STARS here but really the STARS themselves, right? I liked the previous version better.

Reply:
Correct : )